### ISSUE TITLE
Plans to update with "modern" CSS properties

### ISSUE THREAD

**Issue by stowball** [brandon-rhodes/Concentric-CSS#3]
This system is missing many newer CSS(3) properties that are very common, so it would be great for you to document them all with where you would put them

**Comment by brandon-rhodes** [comment]
I agree! Could you provide a link or list of properties that I should be sure to include?

**Comment by sompylasar** [comment]
Here's the list: https://github.com/sasstools/sass-lint/blob/develop/lib/config/property-sort-orders/recess.yml

**Comment by dantesolis** [comment]
hey guys is anyone working on this? Could I take it?

**Comment by brandon-rhodes** [comment]
There does appear to be an outstanding pull request. Let me take a look at it.

**Comment by brandon-rhodes** [comment]
All right:

1. I accepted the pull request.
2. I have just landed a diff that keeps the old `style.css` file my original list of CSS2 properties, and makes `style3.css` the expanded list with recommendations for where to put CSS3 properties.

So, I am open to further improvements or ideas, but let’s keep `style3.css` where we add all the new properties that keep being added, to keep the essential core idea still easy to read through quickly in `style.css`.
